### PR TITLE
Added descriptions and fastrtps sync plots

### PR DESCRIPTION
**Pull request by ahcorde**
Wait this PR https://github.com/ros-infrastructure/ros_buildfarm/pull/740 to merge 

Descriptions for each one of the plots

### CODE REVIEW

File: dashing/ci-nightly-performance.yaml [hunk 1]
```diff
@@ -114,6 +114,7 @@ show_images:
 show_plots:
   Node Spinnig Results:
   - title: Node Spinning Virtual Memory
+    description: "The figure shown above shows the virtual memory usage in Mb used by a single node spinning. All DDS vendors are configured in asyncronous mode except cycloneDDS which is configured in syncronous mode. &lt;br&gt;&lt;/br&gt;&lt;ul&gt;&lt;li&gt;&lt;b&gt;Runtime&lt;/b&gt; 30 seconds&lt;/li&gt;&lt;li&gt;&lt;b&gt;N process&lt;/b&gt; 2&lt;/li&gt;&lt;/ul&gt;"
```
Comment (dirk-thomas) [inline dashing/ci-nightly-performance.yaml]: The proper case is "CycloneDDS".

Comment (dirk-thomas) [reply dashing/ci-nightly-performance.yaml]: Spelling: asyncHronous.

Comment (dirk-thomas) [reply dashing/ci-nightly-performance.yaml]: What is all the escaped html supposed to look like? I would argue this string shouldn't need to be escaped - that should happen when the value is being placed into XML in the template instead.

Comment (ahcorde) [reply dashing/ci-nightly-performance.yaml]: I was playing today with the HTML and the only way that I found to do it, it's escaping the HTML tags

https://wiki.jenkins.io/display/JENKINS/Jelly+and+XSS+prevention

Comment (dirk-thomas) [reply dashing/ci-nightly-performance.yaml]: Sure, but the escaping can happen in the Python logic and doesn't have to be in the config file.

**Review by dirk-thomas — APPROVED** [review]

**Pull request merged**